src/test/regress/expected/multi_connection_cache.out

```diff
@@ -104,3 +108,11 @@ SELECT get_and_purge_connection('localhost', :worker_port);
 (1 row)
 
 SET client_min_messages TO DEFAULT;
+\c
+-- purge existing connection to localhost
+SELECT connect_and_purge_connection('localhost', :worker_port);
```

Comment:
I think doing this test last doesn't actually reproduce the issue at hand here, does it? Because the hash is already created...